src/styles/Login.module.css

```diff
@@ -0,0 +1,58 @@
+.wrapper {
+  color: white;
+  background:
+    linear-gradient(rgba(150, 191, 1, 0.8), rgba(150, 191, 1, 0.8)),
+    url("https://images.unsplash.com/photo-1601871925712-3c803d50fa72?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8YmVhY2glMjB2YWNhdGlvbnxlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&w=1000&q=80");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  opacity: 0.7;
+  background-color: rgba(0, 0, 0, 0.7);
+  margin: auto auto;
+  padding: 40px;
+  border-radius: 5px;
+  box-shadow: 0 0 10px #000;
+  width: 90%;
```

Comment:
I believe that lines 9, 10, 11, and 15 can be removed so your mobile view will look better and the color of the background will become greenish and closer to the original layout.

Reply:
Thanks @ShinobiWarior for your suggestions. I named the file Component.module.css because I want to create a CSS module that only applies to this component. This way, when there's a naming collision with other CSS files or in App.css, no error will happen. You can have a look at this approach [here](https://bootcamp.uxdesign.cc/using-css-modules-in-react-cc17f7c81247)

Reply:
Hi 

> I believe that lines 9, 10, 11, and 15 can be removed so your mobile view will look better and the color of the background will become greenish and closer to the original layout.

Thanks. I've deleted them.